### ISSUE TITLE
TalkBack Reads Incorrect Label for the Account Switching Button in the Navigation Menu

### ISSUE THREAD

**Issue by VoidCrater** [thunderbird/thunderbird-android#7322]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

6.603

### Where did you get the app from?

Google Play

### Android version

Android 12

### Device model

Nokia 5.3

### Steps to reproduce

1. With TalkBack turned on, double tap on "navigate up" (navigation menu) button on the inbox screen 
2. TalkBack should focus on the first item, which is the button to expand the email accounts list

### Expected behavior

TalkBack should have read the expand button as "expand accounts list" or something similar.

### Actual behavior

TalkBack reads the button as the name of the currently selected email account. For example, if I have selected xyz@galaxy.com, the button in the navigation  menu to switch email accounts also reads "xyz@galaxy.com".

**Comment by joohnq** [comment]
I think this issue could be closed, due to the update on the drawer. The drawer doesn't have this button anymore.
